cpu/sam0_common/periph/i2c.c

```diff
@@ -35,11 +35,21 @@
 
 #define SAMD21_I2C_TIMEOUT  (65535)
 
+#define CPU_MODEL_SAML21 (CPU_MODEL_SAML21E18A || CPU_MODEL_SAML21G18A || \
```

Comment:
I think you don't need this define, you can use the predefined macro `CPU_FAM_SAML21` instead.